pkgs/by-name/ss/ssh3/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, libxcrypt
+}:
+
+buildGoModule rec {
+  pname = "ssh3";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "francoismichel";
+    repo = "ssh3";
+    rev = "v${version}";
+    hash = "sha256-0bd2hdvgapTGEGM7gdpVwxelN5BRbmdcgANbRHZ/nRw=";
+  };
+
+  buildInputs = [
+    libxcrypt
+  ];
+
+  vendorHash = "sha256-ZtKxAKNyMnZ8v96GUUm4EukdIJD+ITDW9kHOez7nYmg=";
+
+  ldflags = [ "-s" "-w" ];
+
+  postInstall = ''
+    mv $out/bin/client $out/bin/ssh3
+    mv $out/bin/server $out/bin/ssh3-server
```

Comment:
On darwin, only the client is supported so far and the build currently fails with:
```
Building subPackage ./cli/server
# ssh3/util/linux_util
util/linux_util/linux_user.go:32:10: fatal error: 'shadow.h' file not found
#include <shadow.h>
```
Could you disable the server subpackage on `isDarwin`?

Reply:
If you can provide me with a patch to do so, yes.

Reply:
I think this issue is linked: https://github.com/francoismichel/ssh3/pull/56

Reply:
Nice, yes, that's probably a better solution than extracting the client build today.